stories/tree/__snapshots__/tree.stories.storyshot

```diff
@@ -1,959 +1,1104 @@
-                
-                        
-              </nav>
-              
+                Level 2
+              </span>
+              
                     
             </div>
             
                 
           </div>
           
             
+        </li>
+        
+        
+      </ul>
+      
+    
+    </li>
+    
+    
+    <li
+      aria-level="1"
+      class="fd-tree__item"
+      role="treeitem"
+    >
+      
+        
+      <div
+        class="fd-tree__item-container"
+      >
+        
+            
+        <div
+          class="fd-tree__content"
+        >
+          
+                
+          <span
+            class="fd-tree__text"
+          >
+            Level 1
+          </span>
+          
+            
+        </div>
+        
+    
+      </div>
+    </li>
+    
+    
+    <li
+      aria-level="1"
+      class="fd-tree__item"
+      role="treeitem"
+    >
+      
+        
+      <div
+        class="fd-tree__item-container"
+      >
+        
+            
+        <div
+          class="fd-tree__content"
+        >
+          
+                
+          <span
+            class="fd-tree__text"
+          >
+            Level 1 with very long text that does not wrap ... perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto beatae vitae dicta sunt explicabo. Nemo enim ipsam voluptatem quia voluptas sit aspernatur aut odit aut fugit, sed quia consequuntur magni dolores eos qui ratione voluptatem sequi nesciunt.
+          </span>
+          
+            
+        </div>
+        
+        
+      </div>
+      
+    
+    </li>
+    
+    
+    <li
+      aria-level="1"
+      class="fd-tree__item"
+      role="treeitem"
+    >
+      
+        
+      <div
+        class="fd-tree__item-container"
+      >
+        
+            
+        <div
+          class="fd-tree__content fd-tree__content--wrap"
+        >
+          
+                
+          <span
+            class="fd-tree__text"
+          >
+            Level 1 with very long text and wrapping behaviour ... perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto beatae vitae dicta sunt explicabo. Nemo enim ipsam voluptatem quia voluptas sit aspernatur aut odit aut fugit, sed quia consequuntur magni dolores eos qui ratione voluptatem sequi nesciunt. Neque porro quisquam est, qui dolorem ipsum quia dolor sit amet, consectetur, adipisci velit, sed quia non numquam eius modi tempora incidunt ut labore et dolore magnam aliquam quaerat voluptatem.
+          </span>
+          
+            
         </div>
         
         
       </div>
       
+    
+    </li>
+    
+
+  </ul>
+  
+
+</section>
+`;
+
+exports[`Storyshots Components/Tree Tree With Expanded Level 2 And Borderless Design 1`] = `
```

Comment:
I would rename this to **Tree Level 2 (Borderless)** to simplify it

Reply:
The naming of the sections comes from `export const treeWithExpandedLevel2AndBorderlessDesign = () =>...`. This will translate to  "Tree With Expanded Level 2 And Borderless Design". Unfortunately we cannot put () in the const `treeWithExpandedLevel2AndBorderlessDesign`

Reply:
Oh boy, okay. So then maybe "**Tree Level 2 without Borders**"
and then same can be applied with the others: 
"**Tree Level 6 with Active Items**"
"**Tree Level 3 with Highlight Indicator**"
"**Tree Level 6 with Navigation Links**"